adding check-instances.sh and avoid dup ips while using add-access.sh

llecca opened the pull request.

MarkRoss-Eviden commented:
@llecca thanks for the PR.  I've modified the permissions of the new file as it wasn't executable and tested both that and your amendment to add-access, all looks good

Please can you update your PR to add to the readme file to explain add access won't create a stack if that ip already exists (lines 91-93) and add a new entry before image builder (line 101) to explain the purpose of the new script you've added please?

llecca commented:
> @llecca thanks for the PR. I've modified the permissions of the new file as it wasn't executable and tested both that and your amendment to add-access, all looks good
> 
> Please can you update your PR to add to the readme file to explain add access won't create a stack if that ip already exists (lines 91-93) and add a new entry before image builder (line 101) to explain the purpose of the new script you've added please?

Hi Mark, Please check my updates and revert with your feedback if needed. Also I found a typo which I corrected.
